propertysuggester/parser/JsonReader.py

```diff
@@ -35,9 +35,9 @@ def read_json(input_file):
     count = 0
     for jsonline in input_file:
         count += 1
+        jsonline = str(jsonline, 'utf-8')
```

Comment:
I find this conversion here suspicious. Why is this happening here, line by line? Why isn't the `input_file` read in with UTF-8 being considered?

Reply:
When reading a file in python3, it will be bytes type and needs explicit casting to str type, otherwise does crazy things: http://python3porting.com/problems.html